Control coverage/Feature/CognitiveSearch.md

```diff
@@ -0,0 +1,108 @@
+# Cognitive Search
+**Resource Type:** Microsoft.Search/searchServices
+
+<!-- TOC -->
+
+- [Azure_IoTHubs_Audit_Enable_Diagnostic_Settings](#azure_iothubs_audit_enable_diagnostic_settings)
+
+<!-- /TOC -->
+<br/>
+
+___ 
+
+## Azure_AISearch_AuthZ_Enable_Role_Based_API_Access_Only
+ 
+
+### Display Name 
+Protect Azure AI Search Instances by only allowing RBAC API Access
+
+### Rationale 
+Disabling key-based API access control mitigates the risk of unauthorized access, ensuring that only authenticated users with appropriate credentials can interact with the service. This security measure aligns with best practices, protecting sensitive data and system integrity.
```

Comment:
Please take a separate note, rationale to be updated.

Reply:
Done.